core/templates/css/oppia.css

```diff
@@ -3324,6 +3331,11 @@ md-card.preview-conversation-skin-supplemental-card {
   max-width: none;
 }
 
+.oppia-info-card-tag-row {
+  display: flex;
+  flex-direction: row;
+}
+
```

Comment:
Seems like a duplicate of `fx-row`?

Reply:
Thanks for pointing out! Made the corrections. Please have a look if it is okay now?